src/App.tsx

```diff
@@ -26,70 +27,114 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  switch (sortType) {
+    case (SortType.ALPHABET):
+      visibleGoods.sort((g1, g2) => g1.localeCompare(g2));
+      break;
+
+    case (SortType.LENGTH):
+      visibleGoods.sort((g1, g2) => g1.length - g2.length);
```

Comment:
follow DRY, you can sort just once &  use switch case in callback

Reply:
done 💯